### ISSUE TITLE
Airlines: flight tracker not showing all flights happening today

### ISSUE THREAD

**Issue by whmountains** [duckduckgo/zeroclickinfo-spice#2000]
Expected behavior: show all four "UA1567" flights happening today:
* DEN -> GEG (left yesterday but arrived today)
* IAH -> LAX
* LAX -> DEN
* DEN -> GEG

Actual Behavior: flight tracker only shows first two flights:
* DEN -> GEG
* IAH -> LAX

Interestingly enough, both Flightstats themselves and also Google show the last three flights but miss the first one.  They probably query the flight database based on departure date rather than arrival but that's a topic for another time.

![duckduckgo_flight_error](https://cloud.githubusercontent.com/assets/8680147/8530287/f4c5962a-23d4-11e5-9498-0c39c65ff472.jpg)
![google_flight_info](https://cloud.githubusercontent.com/assets/8680147/8530294/027afa30-23d5-11e5-978d-4f397d039e80.jpg)

------
IA Page:  http://duck.co/ia/view/airlines

**Comment by moollaza** [comment]
@whmountains thanks for reporting this! It's very helpful :+1: 

@tommytommytommy I thinking we'll should increase the window when checking for flights?

**Comment by tommytommytommy** [comment]
Thanks for the heads up, @whmountains and @moollaza. Right now, we show flights based on arrival times for a window 6 hours in the past and 18 hours ahead. Judging by the times shown above, we should've been able to get all the flights because they all fall within 12 hours of each other. But, we don't explicitly state that we want 24 hours worth of data, so perhaps FlightStats is returning less to us. Let's see if specifying a 24 hour window gets us more results. I'm going to wait for this flight to cycle around again next week and test before submitting a PR.

**Comment by tommytommytommy** [comment]
I took a look at this, and I think there are two complicating factors...the first is it seems that the FlightStats API only returns results for one day (not a 24 hour rolling window starting from the time that we specify--specifying the time and a window only allows you to choose a subset of the current day's results), and the second is that the UTC day and the local time day may be different.

I don't believe that widening the window will get us all the results above with the current API call. We need to figure out another way to get that information, either by making a different API call with a rolling window. I took a look at the docs and didn't find such a function, but worth a second pair of eyes exploring and/or contacting FlightStats. We could also make multiple API calls, but it gets pretty awful (and FlightStats probably would not like this) because we'd have to make three calls to consider night time flights from the previous day,  flights for the current day, and early flights for the next day...
